Dockerfile

```
@@ -1,10 +1,21 @@
 FROM tensorflow/tensorflow:1.15.4-gpu-py3
 
-RUN apt-get update && apt-get install -y vim git\
+RUN apt-get update && DEBIAN_FRONTEND="noninteractive" apt-get -y install tzdata && apt-get install -y vim git && apt-get install -y pciutils && apt-get install -y cpio && apt-get install -y wget && apt-get install -y tar && apt-get install -y python-pandas \
```

```suggestion
ENV DEBIAN_FRONTEND "noninteractive"

RUN apt update && apt -y install tzdata vim git pciutils cpio wget tar python-pandas \
```

Also move Line 9 to here

---

Done.